packages/controllers/src/utils/ViemUtil.ts

```diff
@@ -0,0 +1,42 @@
+import { type Chain as ViemChain } from 'viem'
+import { createPublicClient, http } from 'viem'
+import * as viemChains from 'viem/chains'
```

Comment:
very careful with where we import this file. It's pulling in ALL networks so whatever file imports this will import all of the network list (which is HUGE)